plugin/discovery/get.go

```diff
@@ -93,6 +104,18 @@ func (i *ProviderInstaller) Get(provider string, req Constraints) (PluginMeta, e
 	// take the first matching plugin we find
 	for _, v := range versions {
 		url := providerURL(provider, v.String())
+
+		if !i.SkipVerify {
+			sha256, err := getProviderChecksum(provider, v.String())
+			if err != nil {
+				return PluginMeta{}, nil
+			}
+
+			if sha256 != "" {
+				url = url + "?checksum=sha256:" + sha256
```

Comment:
Cool, okay. I think this is non-obvious enough that it's worthy of a comment, since this syntax _looks_ like regular query string arguments that would be sent to the server, and it's not obvious from this point in the code that `go-getter` is going to do further wrangling, particularly if (like me) you don't know that `go-getter` has this feature!